planning/behavior_path_planner/src/utils/geometric_parallel_parking/geometric_parallel_parking.cpp

```diff
@@ -22,7 +22,6 @@
 #include "tier4_autoware_utils/math/unit_conversion.hpp"
 
 #include <interpolation/spline_interpolation.hpp>
-#include <lanelet2_extension/utility/message_conversion.hpp>
 #include <lanelet2_extension/utility/utilities.hpp>
 
 #ifdef ROS_DISTRO_GALACTIC
```

Comment:
(optional) I think we completely removed support for `galactic`. So we can probably remove these conditions as way.

Reply:
Sure we can remove that !

Reply:
It is fine to do it in another PR. I do not think it impacts the build time.